src/api/handler.rs

```diff
@@ -20,6 +20,7 @@ fn init_rpc_handler() -> IoHandler {
     // (specifically security needs to be added)
     // handler.add_method("UpdateArtifacts", update::UpdateArtifacts);
     handler.add_method("GetTests", GetTests);
```

Comment:
ideally `GetAllTests` logic would be implemented as a paramter in GetTests instead of it's own method. I'll accept this as an MVP though.